### PR TITLE
Bug 711932 fix broken addons

### PR DESCRIPTION
**Pull request by zalun**
get_latest is now fixing or deleting a broken add-on

### CODE REVIEW

**Comment by pennyfx** [comment]
This solution looks like it will work.  Could this have been fixed with a SQL script that only updates the broken packages?

**Comment by zalun** [comment]
I'd actually want to concentrate ion finding the issue broken add-ons exist

File: apps/jetpack/tests/test_views.py [hunk 1]
```diff
@@ -2,16 +2,17 @@
 import commonware
 import json
 
-from test_utils import TestCase
 from nose.tools import eq_
 #from nose import SkipTest
-from mock import patch
+from mock import patch   #, Mock
```
Comment (seanmonstar) [inline apps/jetpack/tests/test_views.py]: commented out Mock?

Comment (zalun) [inline apps/jetpack/tests/test_views.py]: yes - I tried it to block the signal - to create a broken add-on

Comment (seanmonstar) [inline apps/jetpack/tests/test_views.py]: I mostly mean that it's confusing after the fact. Uncomment it if it's 
staying, or delete it if it's not.

**Pull request merged**